path_oidc.go

```diff
@@ -284,6 +354,15 @@ func (b *jwtAuthBackend) authURL(ctx context.Context, req *logical.Request, d *f
 		return resp, nil
 	}
 
+	// If configured for form_post, redirect directly to Vault instead of the UI,
```

Comment:
Are there circumstances where perhaps the UI is disabled, and we want to support the 'cli jwt/oidc' login flow, would this break that?